### DISCUSSION TITLE
[已解决]XTLS Vision 在服务端增加一个参数，客户端没选 指纹模拟 拒绝连接

### DISCUSSION THREAD

**Discussion by chika0801** [XTLS/Xray-core#1773] [Q&A]
能否给XTLS Vision在服务端增加一个参数，如果开了，检查到客户端是go指纹就拒绝连接，返回提示让客户端选 指纹模拟 参数。

需要场景是，一个VPS给几个朋友在用，怕朋友忘记选 指纹模拟 选项。

服务端有了这参数，如果朋友连不上了，自然会来找我问。

**Comment by RPRX** [comment]
REALITY 解决了很多问题，其中就包括，客户端必须开指纹模拟，不能用 Go 的指纹，所以加个 REALITY 就行了，一并解决服务端指纹问题

如果你要用自己的域名，现在推荐的是 REALITY 在前，直连更安全，Nginx 在后处理 TLS，还能分流个 gRPC 给 CDN 备用

**Comment by chika0801** [comment]
谢谢帮助，已解决
